dirsrvtests/tests/suites/healthcheck/healthcheck_test.py

```diff
@@ -455,6 +455,36 @@ def test_healthcheck_database_not_initialized(topology_no_sample):
     run_healthcheck_and_flush_log(topology_no_sample, standalone, RET_CODE, json=True)
 
 
+def test_healthcheck_disabled_suffix(topology_st):
+    """Test that we report when a suffix is disabled
+
+    :id: 49ebce72-7e7b-4eff-8bd9-8384d12251b4
+    :setup: Standalone Instance
+    :steps:
+        1. Disable suffix
+        2. Use HealthCheck without --json option
+        3. Use HealthCheck with --json option
+    :expectedresults:
+        1. Success
+        2. HealthCheck should return code DSBLE0002
+        3. HealthCheck should return code DSBLE0002
+    """
+
+    RET_CODE = 'DSBLE0002'
+
+    be_insts = Backends(topology_st.standalone).list()
+    for be in be_insts:
+        be_suffix = ensure_str(be.get_attr_val_utf8_l('nsslapd-suffix')).lower()
+        if be_suffix == DEFAULT_SUFFIX:
+            be.replace("nsslapd-state", "disabled")
+
+    run_healthcheck_and_flush_log(topology_st, topology_st.standalone, RET_CODE, json=False)
```

Comment:
Okay, sorry, I looked at the wrong test results thinking that everything is passed...

Now I see that this one failed in `run_healthcheck_and_flush_log` function:
```
>           assert topology.logcap.contains(searched_code)
E           AssertionError: assert False
E            +  where False = <bound method LogCapture.contains of <LogCapture (NOTSET)>>('DSBLE0002')
E            +    where <bound method LogCapture.contains of <LogCapture (NOTSET)>> = <LogCapture (NOTSET)>.contains
E            +      where <LogCapture (NOTSET)> = <lib389.topologies.TopologyMain object at 0x7fb96d8e4d00>.logcap
dirsrvtests/tests/suites/healthcheck/healthcheck_test.py:55: AssertionError
```

Maybe it's my setup somehow but please, check if it works for you.

P.S. Fedora 32 - 389-ds-base built from your PR - PYINSTALL=true

Reply:
It was a bug in the script - fixed!